register-autonomoustuff-repository/README.md

```diff
@@ -0,0 +1,23 @@
+# Register AutonomouStuff repository
+
+## Inputs
+
+- rosdistro
+  - ROS distribution to register into rosdep
+
+## Outputs
+
+- None
+
+## Sample Workflow Steps
+
+```yaml
+- uses: actions/checkout@v2
+  with:
+    fetch-depth: 0
+
+- uses: autowarefoundation/autoware-github-actions/register-autonomoustuff-repository@tier4/proposal
+  id: register-autonomoustuff-repository
```

Comment:
Is this `id` necessary? Since there is no output, I think it's not.

Reply:
Updated readme at https://github.com/autowarefoundation/autoware-github-actions/pull/6/commits/780e234cde44e98091d99f44456f1198e9b3f0a1.